### PR TITLE
Minor updates

### PR DESCRIPTION
**Pull request by CasperWA**
Some minor updates to the files and the pages.

#### Visual/UX changes
- Add citing header on index (About) page (45d930b).
- Add extra documentation link (in bold) pointing to Swagger/OpenAPI using v1.0.0 of the specification (4d86df4).
- Add a header to the Documentation page (4d86df4).
- Logo links to index (About) page (1a5b41c).
- All external links (in MarkDown files) has the `target` attribute set to `"_blank"`, opening them in a new tab (295dcb8).

#### Development changes
- One sentence per line philosophy implemented for the index (About) page (45d930b).
- Use HTTPS for the link to the webpage in the README, and fix some MarkDown syntax/linting (de1cbf8).

### CODE REVIEW

**Review by ml-evs — APPROVED** [review]
Tested locally and LGTM. Cheers @CasperWA.

**Comment by CasperWA** [comment]
> Tested locally and LGTM.

Cheers!
I've just added a final thing: Making sure all external links open in a new tag (at least the ones defined in the MarkDown pages).
I have nothing more for this PR, and only need the "go-ahead" from @gmrigna, as the main "maintainer" of this site.

**Comment by CasperWA** [comment]
Updated this with the new changes. Please re-review.

File: index.md
Comment (ml-evs) [inline index.md:32]: Noticed a typo from my PR!

```suggestion
All of our work is openly available under the [Materials-Consortia](https://github.com/Materials-Consortia/){:target="_blank"} organization on GitHub and we welcome all contributions.
```

Comment (ml-evs) [reply index.md:32]: Feel free to merge after fixing without re-review. Thanks @CasperWA!

**Review by gmrigna — APPROVED** [review]

**Comment by ml-evs** [comment]
Merging on behalf of @CasperWA, thanks again!

**Pull request merged**
